packages/codegen-ui/package.json

```diff
@@ -18,7 +18,8 @@
     "build:watch": "npm run build -- --watch"
   },
   "dependencies": {
-    "yup": "^0.32.11"
+    "yup": "^0.32.11",
+    "@aws-amplify/ui-react": "^2.20.0"
```

Comment:
It's being used in this file 
`packages/codegen-ui/lib/__tests__/common-component-renderer.test.ts`
and the linter was complaining about it.

Reply:
Since this test seems more focused on the studio component layout, could we make the Base Props any? It should work independent of what props are being passed in.

Reply:
Changed test to remove dependency